modules/renter/download.go

```diff
@@ -217,6 +217,10 @@ func (d *download) fail(err error) {
 	d.downloadComplete = true
 	d.downloadErr = err
 	close(d.downloadFinished)
+	closeErr := d.destination.Close()
+	if closeErr != nil {
+		build.Severe("error closing download destination: " + closeErr.Error())
```

Comment:
We'd probably just log the error. If we don't have access to a logger, we can just write a TODO or something - this code is going to get mixed up during our 15 sprints for v1.3.2. I didn't realize that we're already in the fail function. build.Severe is definitely incorrect, because an error here can happen that is not a fault of the programmer nor a reason that the user cannot continue with future actions.

Sorry about that.